pkgs/by-name/si/silverbullet/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  deno,
+}:
+stdenv.mkDerivation (
+  finalAttrs: {
+    pname = "silverbullet";
+    version = "0.7.3";
+
+    src = fetchurl {
+      url = "https://github.com/silverbulletmd/silverbullet/releases/download/${finalAttrs.version}/${finalAttrs.pname}.js";
```

```suggestion
      url = "https://github.com/silverbulletmd/silverbullet/releases/download/${finalAttrs.version}/silverbullet.js";
```

interpolating `finalAttrs.pname` can cause issues when overriding the package name.